src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/rc_get_capabilities_response_test.cc

```diff
@@ -138,7 +138,7 @@ TEST_F(RCGetCapabilitiesResponseTest, RUN_SUCCESSS) {
   radio_control_capability["radioFrequencyAvailable"] = true;
   radio_control_capability["hdChannelAvailable"] = true;
   radio_control_capability["rdsDataAvailable"] = true;
-  radio_control_capability["availableHDsAvailable"] = true;
```

Comment:
I would leave availableHDs available in the tests for now. It is deprecated so it can technically still be used even though it is not recommended. Functionality should still exist for the parameter.

Reply:
@JackLivio Thanks. Fixed in be2dd5badfc9fef8734b56fce1a1827f8ac1aee0